src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="paje"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class name 'page' is better than 'paje'.